README.md

```diff
@@ -90,4 +90,8 @@ $ docker build --target production --tag todo-app:prod .
 Run the image:
 ```bash
 $ docker run --env-file ./.env -p 80:80 --mount type=bind,source="$(pwd)"todo_app,target=/app/todo_app todo-app:prod
-```
\ No newline at end of file
+```
+
+### CD pipeline:
+If the build completes and the tests pass, the app will be automatically deployed to Heroku. 
+The url for the app is: https://peter4137-todo-app.herokuapp.com/
```

Comment:
Thanks for documenting this - using the name as a secret would have made this hard otherwise!